client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -55,19 +55,11 @@ class PeopleListSectionHeader extends Component {
 		const { currentRoute, translate } = this.props;
 
 		if ( startsWith( currentRoute, '/people/followers' ) ) {
-			// Adding a dot at the end of the translation. New translation for RTL languages.
-			return i18n.hasTranslation( 'A list of people currently following your site.' )
-				? translate( 'A list of people currently following your site.' )
-				: translate( 'A list of people currently following your site' ) + '.';
+			return translate( 'A list of people currently following your site.' );
```

Comment:
Thank you! @danielbachhuber .

I've approved the pending translations:
<img width="433" alt="Screenshot 2023-01-16 at 15 57 09" src="https://user-images.githubusercontent.com/779993/212720493-e53f383e-6525-43b2-9e07-27f26c74c660.png">

- https://translate.wordpress.com/projects/wpcom/-all-translated/859882/
- https://translate.wordpress.com/projects/wpcom/-all-translated/859883/

Not sure about the RTL languages, though.

We are good to merge this PR, because the translation exists, and `hasTranslation` will return true, showing the latest text.